service-discovery/quickstart.yml

```diff
@@ -1,34 +0,0 @@
-apiVersion: console.openshift.io/v1
```

Comment:
Why this is removed?

Reply:
@wtrocki Removed this file because we are not building this content as a quick start - it is a service binding guide.